cmd/defaults.yaml

```diff
@@ -14,6 +14,44 @@ Tracing:
   Fraction: 1.0
   MetricPrefix: zitadel
 
+Analytics:
+  # Push analytics data to defined endpoints
+  Push:
+    # Push the data to all these endpoints at least once.
+    # If one endpoint returns an unsuccessful response code or times out,
+    # ZITADEL retries to push the data point to all configured endpoints again until it succeeds.
+    # The endpoints can be reconfigured at runtime.
+    # Three redirects are followed.
+    # Configure delivery guarantees and intervals in the section Projections.Customizations.Analytics
+    Endpoints:
+      # Include https://zitadel.com/usage if you want to help the ZITADEL maintainers improve the product's usability by giving them some insights.
+      # Don't forget to opt in by setting AllEnabled to true.
+      - https://zitadel.com/usage
+    # If AllEnabled is true, all data listed below is pushed, regardless if they have the value true or false
+    AllEnabled: false
+    EnabledMetrics:
+      # InstanceCreated is sent when a new virtual instance is created.
+      # It is also emitted, when the first instance is created during the setup phase.
+      # If InstanceCreated is false and AllEnabled is false, then the data points about created instances are omitted.
+      InstanceCreated: false
```

Comment:
Does it make sense to enable every attribute separatly? 
Maybe just make onboarding process metrics?

Reply:
Normally you just set AllEnabled to true. But if it's false, you can control which data is sent with fine granularity.

Should we remove it for now?

Reply:
I'll remove it and move the comments to the docs.

Reply:
done